src/Components/IncomeStatement.js

```diff
@@ -20,28 +20,33 @@ const IncomeStatement = () => {
               <h4>
                 <span>Cost of Revenue</span>
                 {' '}
+                $
                 {state.costOfRevenue}
               </h4>
               <h4>
                 {' '}
                 <span>Cost and Expenses</span>
                 {' '}
+                $
                 {state.costAndExpenses}
               </h4>
               <h4>
                 <span>Gross Profit</span>
                 {' '}
+                $
                 {state.grossProfit}
               </h4>
               <h4>
                 {' '}
                 <span>Net Income</span>
                 {' '}
+                $
                 {state.netIncome}
               </h4>
               <h4>
                 <span>Revenue</span>
                 {' '}
+                $
                 {state.revenue}
               </h4>
```

Comment:
- [ ] The details page works as expected, but the design doesn't match the original one, kindly take a look and make the changes based on the given design.

![finance](https://user-images.githubusercontent.com/53124548/217782980-3baf9e58-1a9d-4056-913c-66a60eeaebf8.png)

Reply:
Hello @KDlamini 
  Sorry for the disturb so if you did not submit the review can you pull it and check this part is revolved or not 
 
Thank you

Reply:
@Hassaanjbaig-code Well resolved ✔️